src/server/game/Entities/Player/PlayerStorage.cpp

```diff
@@ -5027,7 +5027,7 @@ bool Player::LoadFromDB(ObjectGuid playerGuid, CharacterDatabaseQueryHolder cons
     SetFloatValue(UNIT_FIELD_HOVERHEIGHT, 1.0f);
 
     // load character creation date, relevant for achievements of type average
-    SetCreationDate(fields[74].Get<time_t>());
+    SetCreationTime(fields[74].Get<Seconds>());
```

Comment:
Unsure if this actually works. `UNIX_TIMESTAMP(creation_date)` is what you get

Reply:
On my windows setup this works as expected, m_creationTime has the correct Unix timestamp in seconds after this call